cmd/sidecars/network-passt-binding/domain/configurator_test.go

```diff
@@ -98,7 +98,7 @@ var _ = Describe("pod network configurator", func() {
 				&vmschema.Interface{Name: "default", Binding: &vmschema.PluginBinding{Name: "passt"}},
 				&domainschema.Interface{
 					Alias:       domainschema.NewUserDefinedAlias("default"),
-					Type:        "user",
+					Type:        "vhostuser",
```

Comment:
the repetition might worth using a const in this file, or even shared with the core ?